corehq/apps/export/forms.py

```diff
@@ -1115,4 +1115,7 @@ def __init__(self, domain, *args, **kwargs):
 
     @staticmethod
     def domain_datasources(domain):
-        return [(ds.data_source_id, ds.display_name) for ds in get_datasources_for_domain(domain)]
+        return [
+            (ds.data_source_id, ds.table_id)
```

Comment:
Do you think adding the table name next to the ID `my_id_123 (table_name)` could be useful to users?

Reply:
I wanted to do that and checked with Livhuwani, but she mentioned that it's not necessary since the data sources created by users will always have legible names, so she suggestioned that we stick to only the `table_id`.

Reply:
Ah okay, well in that case it is fine. I just wanted to make sure this was considered. Thanks for clarifying!

Reply:
I believe the ID is never visible to the user and is only used in URL or internally.

Reply:
Another more involved solution could be to keep to the display name, but only show the table name in brackets for duplicates